pre/remap_restart/remap_utils.py

```diff
@@ -17,8 +17,8 @@
 
 #During cmake step, the string will be changed according to the system
 
-BUILT_ON_SLES15 = "@BUILT_ON_SLES15@"
-GEOS_SITE       = "@GEOS_SITE@"
+BUILT_ON_SLES15 = "TRUE"
```

Comment:
We might not even need `BUILT_ON_SLES15` anymore if all it handles is the `scutest` bits.

Now `GEOS_SITE` we do *not* want to hardcode to NCCS. It should stay as `@GEOS_SITE@` as I think we do different things if at different sites (i.e., NAS with PBS) though...that might just be future hope?

Reply:
that was a mistake, I revert it :)